backend/src/main/java/gov/cdc/usds/simplereport/db/model/auxiliary/RequestVariablesSerializer.java

```diff
@@ -0,0 +1,42 @@
+package gov.cdc.usds.simplereport.db.model.auxiliary;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.nimbusds.jose.util.StandardCharset;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+import javax.servlet.http.Part;
+
+public class RequestVariablesSerializer extends JsonSerializer<Map<String, Object>> {
+
+  private Object parseValue(Object value) {
+    if (value instanceof Part) {
+      Part part = (Part) value;
+      try {
+        return new BufferedReader(
```

Comment:
I would like to think we don't need to do a line-by-line read and then rejoin into a bigger string again—granted that Java readers and streams are annoying, it seems like we should be able to read the whole thing in at one go. Then again, we could just leave it as a `List<String>` for serialization...